docs/docs/deployments.md

```diff
@@ -18,6 +18,97 @@ Multiple deployments can be performed at the same, as long as one application is
 If a deployment is requested, that tries to change an application that is already being changed by another active deployment, 
 the new deployment request will be rejected.
 
+## Basics
+
+Let us start with a simple example of a deployment: an app that prints `Hello Marathon` to stdout and then sleeps for 5 sec, in an endless loop.
+You would use the following application resource (in JSON format) to describe the deployment: 
+
+```json
+{
+    "id": "basic-0", 
+    "cmd": "while [ true ] ; do echo 'Hello Marathon' ; sleep 5 ; done",
+    "cpus": 0.1,
+    "mem": 10.0,
+    "instances": 1
+}
+```
+
+Note that `cmd` in the above example is the command that gets executed. 
+Its value is wrapped by the underlying [Mesos fetcher](https://github.com/apache/mesos/blob/master/src/launcher/fetcher.cpp) via `/bin/sh -c ${cmd}`.
```

Comment:
The command is actually being wrapped by the executor, the fetcher only fetches the resources required to execute the command.